srtcore/core.cpp

```diff
@@ -6257,10 +6257,10 @@ static void DebugAck(string hdr, int prev, int ack)
     }
 
     prev = CSeqNo::incseq(prev);
-    int diff = CSeqNo::seqcmp(ack, prev);
+    int diff = CSeqNo::seqoff(prev, ack);
```

Comment:
`DebugAck()` is used for heavy logging. The change does not affect SRT functionality.

Reply:
Hit by the general search-replace query, I only skipped replacement when it was clear that the result is only compared against 0. This is for debug purposes, but it better work correctly or should be disabled.